### PR TITLE
Update to Font Awesome

### PR DESCRIPTION
**Pull request by iskra0332**
Okay I am going to go ahead and make the pull request, but I can not get this to work correctly on my site. The discord button is still not showing an icon at my website, and in fact now the github icon is not showing up.

I downloaded the source code for 5.7.2 and copy pasted it into the file and I changed the urls in the @font face by just changing 4.1.0 to 5.7.2. 

I am not sure why it is not working. Discord is in the list so I would think it should work. But I don't really understand much of it.

### CODE REVIEW

**Comment by iskra0332** [comment]
I looked into it more and tried figuring out how to use @font-face and font awesome. I am just not familiar with css and webpage making. Tried googling stuff to no success.

I think the urls are the problem, I used the old urls and just changed the version number, some of the fonts are loading up but some are not, like the github icon even quit working, and no discord icon. You can see at limasecurityworks.io.

I also don't understand what the two periods mean at the start of the old 4.1.0 means like ../fonts/fontawesome-webfont.woff?v=5.7.2 or where those links came from. Again that is the 4.1.0 link and I just tried changing 4.1.0 to 5.7.2 and it does not seem to be working.

Going to keep messing with it later though and see if I can figure it out.

**Comment by iskra0332** [comment]
Upon further investigation, it seems that the issue is that I needed to update the files in the fonts folder too, I downloaded the source code for 5.7.2 and replaced old files in the css/fontawesome/css and the css/fontawesome/fonts folders. I tried running it but it seems the rest of the site is not working with it. It looks like font awesome reorganized how those folders were set up. For example the fonts folder in 5.7.2 no longer just contains 4 files, it is 15 and the css folder now contains 14 instead of 2.

I suspect that names and things were changed where the rest of the css and js files of the site need to be fixed to work with it.

Still working on it.

**Comment by iskra0332** [comment]
Okay, for an update that no one cares to hear. I got the discord icon to work with font awesome's 5.7.2. Going to get everything together and hopefully get everything complete today or tomorrow.

**Comment by iskra0332** [comment]
Okay, I think that I have completed the update. It should be ready for review. I made a commit called Updates for 5.7.2 Font Awesome. Changes are in the _includes folder and the css/font-awesome folders.

Let me know if you have any questions.

File: _includes/css/main.css [hunk 1]
```diff
@@ -54,7 +54,8 @@ hr.star-primary:after {
     position: relative;
     top: -.8em;
     padding: 0 .25em;
-    font-family: FontAwesome;
+    font-family: "Font Awesome 5 Free";
+    font-weight: 900;
```
Comment (jeromelachaud) [inline _includes/css/main.css]: why adding font-weight prop?

File: _includes/head.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
     {% if site.bing_verify %}<meta name="msvalidate.01" content="{{ site.bing_verify }}">{% endif %}
 
     <!-- Custom Fonts -->
-    <link rel="stylesheet" href="{{ "/css/font-awesome/css/font-awesome.min.css" }}">
+    <link href="/css/font-awesome/css/all.min.css" rel="stylesheet">
```
Comment (jeromelachaud) [inline _includes/head.html]: why removing the dynamic link?

File: _includes/css/main.css (hunk 1 above)
Comment (iskra0332) [reply _includes/css/main.css]: Great to hear from you. Really enjoyed working on this and I like the theme. Thank you for the work you have put into it.

As for the font-weight, it asks you to put a font weight when upgrading as you can see from this link.

https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4

I initially missed that part and the stars showed up as a regular star. At the following link I read that you can add font weight to change to the solid star.

https://stackoverflow.com/questions/49766325/fontawesome-5-unicode

I added font-weight 400 and it was still the regular star. I changed it to 900 and got the solid star.

File: _includes/head.html (hunk 2 above)
Comment (iskra0332) [reply _includes/head.html]: Honestly I don't know what a dynamic link is, this is my first time working with html and css, so it is great that you are checking this stuff haha. But I just posted it like they did at this link under additional styling options. 

https://fontawesome.com/how-to-use/on-the-web/setup/getting-started

**Comment by jeromelachaud** [comment]
Thanks again, I'll have a more thorough look at it over the week-end 👍

**Comment by iskra0332** [comment]
Sure thing! Look forward to answering any questions.

**Comment by jeromelachaud** [comment]
https://github.com/jeromelachaud/freelancer-theme/pull/146

**Pull request closed without merging**